### ISSUE TITLE
High CPU utilization in 'development' mode on Windows

### ISSUE THREAD

**Issue by drosen0** [socketstream/socketstream#270]
Recently, node.exe is using 30-60% CPU continuously at idle, only while in SS 'development' mode. I don't think this was an issue previously, but I'm not sure at what point this became an issue.

* Windows 7 32-bit
* NodeJS 0.8.4
* SocketStream master

I haven't tested other platforms or versions.

**Comment by paulbjensen** [comment]
Hi @drosen0, are you still experiencing this issue. If not, can I close the issue?

**Comment by drosen0** [comment]
Hi @paulbjensen, thank you for following up on this issue. I haven't used SocketStream in quite a long time (and also I switched development platforms), so I can't comment on whether it's fixed. I do use [grunt-contrib-watch](https://github.com/gruntjs/grunt-contrib-watch), which uses [gaze](https://github.com/shama/gaze). I've noticed that on Mac, some of the projects I work on do use considerable CPU while watching (it seems to be those using an older version of grunt-contrib-watch, and therefore older version of gaze; or it could be that the older projects have more files) and some do not.
